proto/cosmos/crypto/keyring/types.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+package cosmos.crypto.keyring;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/keyring";
+option (gogoproto.goproto_getters_all) = false;
+
+message KeyringEntry {
+  string name = 1;
+  google.protobuf.Any pub_key = 2;
+
+  oneof item {
+    LocalInfo local = 3;
+    LedgerInfo ledger  = 4;
+    OfflineInfo offline  = 5;
+    MultiInfo multi  = 6;
+  }
+}
+
+// Item is a keyring item stored in a keyring backend.
+message LocalInfo {
+  option (gogoproto.goproto_stringer) = false;
+
+  google.protobuf.Any privKeyArmor = 1;
```

Comment:
This is not armored private key. It should be one of the privkey instances wrapped into Any, so:
```
google.protobuf.Any privKey = 1;
```